src/backends/tensorrt/tensorrtlib.cc

```diff
@@ -419,10 +419,19 @@ namespace dd
 
     if (predict_dto->parameters->mllib->gpuid->_ids.size() == 0)
       throw MLLibBadParamException("empty gpuid vector");
+    if (predict_dto->parameters->mllib->gpuid->_ids.size() > 1)
+      throw MLLibBadParamException("multigpu not supported");
```

Comment:
Maybe slightly more explicit, "Tensorrt: multi-gpu inference is not applicable".